src/main/haskell/language-kore/src/Data/Kore/AST/PureToKore.hs

```diff
@@ -56,50 +60,43 @@ extractPurePattern level p =
     (IsObject, IsObject) -> Fix p
     _ -> error ("Undexpected non-" ++ show level ++ " pattern")
 
-attributesPureToKore
-    :: MetaOrObject level => PureAttributes level -> KoreAttributes
-attributesPureToKore ma =
-    Attributes (map patternPureToKore (getAttributes ma))
-
+-- FIXME : all of this attribute record syntax stuff
+-- FIXME: sillyCoerce should not exist for long
```

Comment:
it seems you got rid of coerce.  Could you also remove the corresponding imports?

Reply:
Do this comments still make sense?

Reply:
Removed the "sillyCoerce" fixme, but I'd rather not forget about the other thing. As you can see I had to use some weird syntax to change the types. Data.Coerce can do it, but I think it "does too much" because it also throws an ambiguity error for 'level'